examples/model_assisted_labeling/image_mal.ipynb

```diff
@@ -23,13 +23,7 @@
    "metadata": {},
```

Comment:
<p>instead of this, should we make the API key just this?</p><p> </p><pre class="ql-syntax" spellcheck="false">API_KEY = os.environ.get("LABELBOX_API_KEY")
if not API_KEY:
&nbsp; &nbsp; raise EnvironmentError("Missing API Key")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>